core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -108,33 +108,6 @@ public void before() throws IOException {
         mSessions, mBlockStore, mUfsManager);
   }
 
-  @Test
-  public void openUnderFileSystemBlock() throws Exception {
```

Comment:
Since the logic is moved to `createUfsBlockReader`, make sure the logic is still tested.

Reply:
the test here is covered by UnderFileSystemBlockStoreTest